examples/aditof-test/main.cpp

```diff
@@ -339,13 +347,31 @@ int _DisplayIR(cv::Mat *irMat, int *measuredDistance, int targetDistance,
         }
     }
 
+    /* Read the camera temperature */
+    std::vector<std::shared_ptr<aditof::TemperatureSensorInterface>>
+        tempSensors;
+    camera->getTemperatureSensors(tempSensors);
+    tempSensors[0]->read(*temperature);
```

Comment:
You should check if tempSensors vector in not empty before accessing its first element.